src/common/visualization/visualization.py

```diff
@@ -0,0 +1,81 @@
+# SPDX-FileCopyrightText: 2023 spdx contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+from typing import Dict, List
+
+try:
+    from networkx import DiGraph
+except ImportError:
+    DiGraph = None
+from spdx.document_utils import get_contained_spdx_element_ids, get_element_from_spdx_id
+from spdx.model.document import Document
+from spdx.model.relationship import Relationship
+
+
+def visualize_document(document: Document, visualize: str) -> None:
```

Comment:
Maybe it's just because I haven't used graph libraries in python before, but I find it rather unclear what kind of strings `visualize` would take or what it actually does. (Sounds more like a Boolean to me.)

Reply:
This is the file where the generated graph - or rendered picture depending on the file extension will be written to. I will rename this variable.